uaclient/api/u/security_status/get_package_manifest/v1.py

```diff
@@ -0,0 +1,56 @@
+from apt import Cache  # type: ignore
+
+from uaclient import snap
+from uaclient.api.api import APIEndpoint
+from uaclient.api.data_types import AdditionalInfo
+from uaclient.config import UAConfig
+from uaclient.data_types import DataObject, Field, StringDataValue
+
+
+class InstalledPackagesResults(DataObject, AdditionalInfo):
+    fields = [
+        Field("manifest_data", StringDataValue),
+    ]
+
+    def __init__(self, manifest_data: str):
+        self.manifest_data = manifest_data
+
+
+def get_installed_packages() -> InstalledPackagesResults:
+    return _get_installed_packages(UAConfig())
+
+
+def _get_installed_packages(cfg: UAConfig) -> InstalledPackagesResults:
+    """Returns the status of installed packages (apt and snap packages)
+    Returns a string in manifest format i.e. package_name\tversion
+    """
+    cache = Cache()
+    manifest = ""
+    # get apt packages
+    for package in cache:
+        if package.is_installed:
+            version = [
+                version.version
+                for version in package.versions
+                if version.is_installed
+            ][0]
+            manifest += "{app}\t{version}\n".format(
+                app=package.name, version=version
+            )
```

Comment:
There is always a single installed version, no need to loop through.
```suggestion
            manifest += "{app}\t{version}\n".format(
                app=package.name, version=package.installed.version
            )
```

And I think the implementation with `Cache` is ok, unless you want the optimized shortest delay here. 
I'd say we don't need it, but `dpkg-query -W` will return names and versions just in case.

Reply:
@CalvoM I think we should either:

* modify the apt module function to use cache and control if the output will be with or without the versions
* modify the apt module function dpkg call to include the package version if required.

I would vote for the `dpkg` change since I believe is a faster approach that we are already using

Reply:
I see, then the second option is okay, since it already being used. I will check if I need versions as well.